src/app/server/DataModelHandler.cpp

```diff
@@ -65,7 +65,17 @@ void InitDataModelHandler()
 #endif
 }
 
-void HandleDataModelMessage(NodeId nodeId, System::PacketBufferHandle buffer)
+#ifdef USE_ZAP_CONFIG
```

Comment:
what does the 'zap config' flag represent?

It does not seem intuitive to me why a ACL wouldbe gated by a zap configration.

Reply:
It's mainly because we are passing `EmberApsFrame` as the parameter to the function that checks the ACL. This structure has the ZCL endpoint/cluster etc information, and is only available under USE_ZAP_CONFIG flag. We can probably extract the IDs needed to check ACL, but given that architecture is still evolving, I thought it's better to send a reference to the full structure.

Reply:
I found the header needed to get access to EmberApsFrame. So the ifdef is not needed anymore.